app/_src/gateway/upgrade/backup-and-restore.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Backup and restore
+---
+
+Backup and restore methods described in this post only serve as general instructions. You must revise the methods in line with your infrastructure, deployment, and business requirements.
+
+Before upgrading, you should back up data with database tools and optionally declarative tools. We highly recommend doing backup in both methods, which offers you recovery flexibility. The database tools are robust and can restore data instantly compared to the declarative tools. Consequently, in case of data corruption, try to do a database-level restore first, and otherwise bootstrap a new database and then use declarative tools.
+
+## Database
```

Comment:
Probably a small suggestion, but I think maybe separating the sections by backup and restore vs. database and declarative would work?

## Backup
### database
### declarative
## Restore
### database
### declarative

Just because everyone needs to backup, they might not need to restore, so best to put that at the bottom and separate from backup in case they don't need it?